etc/tendrl/tendrl.conf.sample

```diff
@@ -0,0 +1,17 @@
+[bridge_common]
```

Comment:
conf files are not ideally created/copied by rpm installation methods, We prefer if the admin/tendrl install scripts do it. Lets revisit this if we face any nags in future